src/api/index.ts

```diff
@@ -1,7 +1,7 @@
 type FetchData = Record<string, unknown>
 type FetchOptions = Record<string, string | string[] | Record<string, string>>
 
-const basePath = localStorage.getItem("basePath") || "https://homagix-server.dilab.co"
+const basePath = localStorage.getItem("basePath") || "/api"
```

Comment:
this is of course a quick fix because I was lazy. Can we find a better way to set the basePath than the localStorage?

Reply:
An alternative is to load the frontend assets from the same URL host as the backend API. This would require us to use another asset server than github pages. If we do so, your fix is just the best solution IMO.

We would still be able to use an alternative URL for dev or test purposes, though we would either need to explicitly set it locally for dev, or use a local backend as well.

Actually, using the default URL was a lazy solution from me to make use of github pages easily. We don't really need that, if we deploy the frontend in a way that the backend can serve the built frontend assets. I think I will do so, but with another PR.